lib/index.js

```diff
@@ -84,12 +84,24 @@
 //               elements : [element1, element]
 //           }
 //
-//   Where the timetag is a javascript-native numeric value of the timetag,
-//   and elements is an array of either an _OSC Bundle_ or an _OSC Message_
-//   The `oscType` field is optional, but is always returned by api functions.
+//   `oscType` ("bundle"|"message")
```

Comment:
I don't think this is right - osc bundles have to be bundle.  I would remove this line.